homeassistant/components/prometheus.py

```diff
@@ -31,13 +28,15 @@
 DOMAIN = 'prometheus'
 DEPENDENCIES = ['http']
 
+CONF_FILTER = 'filter'
 CONF_PROM_NAMESPACE = 'namespace'
 PROM_NAMESPACE = 'hass'
 
 CONFIG_SCHEMA = vol.Schema({
-    DOMAIN: FILTER_SCHEMA.extend({
+    DOMAIN: vol.All({
+        vol.Optional(CONF_FILTER, default={}): entityfilter.FILTER_SCHEMA,
```

Comment:
@pvizeli This is what you asked, isn't it? Using the `entityfilter.FILTER_SCHEMA` at this point (instead of the previous other-component-stuff-that-may-break).